src/python/ImpactXParticleContainer.cpp

```diff
@@ -118,10 +118,6 @@ void init_impactxparticlecontainer(py::module& m)
              "Compute reduced beam characteristics like the position and momentum moments of the particle distribution, as well as emittance and Twiss parameters."
         )
 
-        .def("redistribute",
-             &ImpactXParticleContainer::Redistribute,
-             "Redistribute particles in the current mesh in x, y, z"
-        )
```

Comment:
Is this function no longer needed?  Not sure why this is deleted here--how it is related to this PR.

Reply:
Correct. This is actually already exposed in pyAMReX. The binding here overwrote it and, problematically, did not define the proper default arguments. Just removing it solves this and removes the sloppy duplication that I did by accident.

Reply:
Moved to #833